### ISSUE TITLE
How to deal with unclear voice?

### ISSUE THREAD

**Issue by DabiaoMa** [CSTR-Edinburgh/merlin#148]
Hi, all,

I am using Merlin to do transfer learning. Though the generated voice sounds like the voice in the training set, it is not clear. I am thinking about to change the loss, but I know nothing about which acoustic features determine the clearness of the voice. Would you please give me some hints about it?
